CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## master / unreleased
+
+ - `LastCheckpoint` used to return just the segment name and now it returns the full path.
+ - `NewSegmentsRangeReader` can now read over miltiple wal ranges by using the new `SegmentRange` struct.
+ - `CorruptionErr` now also exposes the Segment `Dir` which is added when displayin any errors. 
```

Comment:
```suggestion
 - `CorruptionErr` now also exposes the Segment `Dir` which is added when displaying any errors. 
```

Reply:
Aaargh, when will vscode add a spell check for the comments!